src/gettingstarted/template/create-project.md

```diff
@@ -0,0 +1,49 @@
+# Start with a template
+
+## Create a new project from a template
+
+In the previous step, you set up a free trial account with Platform.sh. Now you have access to the Platform.sh [management console](/administration/web.md).
+
+From here you can create projects, adjust account settings, and a lot more that you will explore throughout these Getting Started guides.
+
+<video controls autoplay loop>
+  <source src="/videos/management-console/create-project-mc.mp4" type="video/mp4">
+</video>
+
+Since you do not yet have any projects on Platform.sh, the first thing you will see when you sign in is a workflow for creating a new project.
+
+1. **Name your project**
+
+   Give your project a name like `My First Project` and then click `Next`.
+
+2. **Set code base**
+
+   Now you will be able to see the large collection of Platform.sh's supported templates. They are simple applications, but they could also be the starting place from which you could build something more interesting.
+
+   If you are more comfortable with a particular language, click the dropdown labeled `All languages`. Select a language and the template list will update.
+
+   Select a template and click `Next`.
+
+3. **Plan size**
+
+   The management console gives you the option of setting your plan size from the very beginning, but you can always upgrade your plan later on as well.
+
+   For now, select the default `Professional Standard` plan size and click `Next`.
+
+4. **Project region**
+
+   The last step for setting up a new project is to configure its region.
```

Comment:
Add: "The region corresponds with the data center where your project will live."